src/current/v25.2/show-policies.md

```diff
@@ -0,0 +1,127 @@
+---
+title: SHOW POLICIES
+summary: The SHOW POLICIES statement lists the row-level security policies for a table
+toc: true
+keywords: security, row level security, RLS
+docs_area: reference.sql
+---
+
+The `SHOW POLICIES` statement lists the [row-level security (RLS)]({% link {{ page.version.version }}/row-level-security.md %}) policies for a [table]({% link {{ page.version.version }}/schema-design-table.md %}).
+
+## Syntax
+
+<!--
+
+NB. This is commented out while we wait for a fix to DOC-12125
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/show_policies_stmt.html %}
```

Comment:
Ready to include?

Reply:
filed a followup issue and added the following comment:

```
NB. This was waiting on a fix to DOC-12125 when this doc was being
written. Now there is additional followup work (tracked in DOC-13653)
to update the parameters and potentially the diagram.
```